README.md

```diff
@@ -170,21 +172,27 @@ facilitate this process.
 
 ## Configuration Options
 
-All configuration options are documented in the [values.yaml](/helm/nginx-ingress-controller-app/values.yaml) file.
+All configuration options are documented in the [values.yaml](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/helm/nginx-ingress-controller-app/values.yaml) file.
 
 ### Internal nginx IC options
 
-This chart contains a template for an additional [internal service](helm/nginx-ingress-controller-app/templates/controller-service-internal.yaml) to cover the use case of having separate internal and external ingress routes with a single nginx IC instance. 
+This chart contains a template for an additional [internal service](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/helm/nginx-ingress-controller-app/templates/controller-service-internal.yaml) to cover the use case of having separate internal and external ingress routes with a single nginx IC instance.
+
+**Important:** On AWS, internal Services require the following prerequisites:
+
+- The cluster is running workload cluster release [v17.3.2](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v17.3.2/) or above
+- At least one node pool [`MachineDeployment`](https://docs.giantswarm.io/ui-api/management-api/crd/machinedeployments.cluster.x-k8s.io/) has been created with `alpha.giantswarm.io/internal-elb: true` annotation
+- Controller pods are scheduled on nodes of the node pool with the above annotation. (For example selecting nodes with matching `giantswarm.io/machine-deployment` label using `controller.nodeAffinity` configuration value)
```

Comment:
@njuettner can you quickly tell me if these instructions point in the right direction?

Also an additional question: Do you know if external aws load balancers on these nodes will also work?

Reply:
Only pods which are exposed via node port need to placed on those machinedeployments. We figured out that kube-controller updates security groups for pods which have a node port.

AFAIK ingress controller doesn't need to on those machinedeployment as long as they don't use node ports for themselves.

Yes that should work as well